hack/Makefile

```diff
@@ -0,0 +1,11 @@
+tests = $(shell ./run_test --list 2>/dev/null)
+
+.PHONY: all
+
+all: $(tests)
+
+%:
+	@if [[ "$(tests)" =~ "$@" ]]; then \
+		echo Running test: $@; \
+		VERSION=$(VERSION) IMAGE_NAME=$(IMAGE_NAME) ./run_test $@; \
+	fi;
```

Comment:
I stlil don't think this needs to be in `Makefile` or `hack/Makefile`.  can not this be directly in common.mk?